actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BaseBulkUploadBackgroundJobActor.java

```diff
@@ -18,7 +17,7 @@ public void setSuccessTaskStatus(
       throws JsonProcessingException {
```

Comment:
BulkProcessStatus - use this enum as data type for status. It will be easier for the caller. 
Set visibility to protected as well.